cms/forms/widgets.py

```diff
@@ -102,10 +102,10 @@ def render(self, name, value, attrs=None):
     var handlePageChange = function(page_id) {
         if (page_id) {
             $("#id_%(name)s_2 option").attr('selected', false);
-            $("#id_%(name)s_2 option[value=" + page_id + "]").attr('selected', true);
+            $("#id_%(name)s_2 option[value='" + page_id + "']").attr('selected', true);
         } else {
             if ($("#id_%(name)s_2").length) {
-                $("#id_%(name)s_2 option[value=]").attr('selected', true);
+                $("#id_%(name)s_2 option[value]").attr('selected', true);
```

Comment:
shouldn't it be `option[value='']`, since we are trying to pick the empty value not every value?